.github/workflows/dotnet-integration-tests.yml

```diff
@@ -8,6 +8,10 @@ on:
   workflow_dispatch:
   push:
     branches: [ "main" ]
+    paths:
```

Comment:
I think this can have the same path ignores as dotnet-ci. The samples have no bearing on the integration tests.

Reply:
This "includes" everything in the dotnet folder, including metadata, csproj, whereas the others "ignore" stuff from certain samples folders. The effective difference is subtle,

Including dotnet ensures any change at all, in the dotnet folder (including weird config file changes, etc), will trigger integration tests.

Excluding the non-C# samples still includes the azure functions sample and runs dotnet tooling, build, format, etc. on all dotnet files.

Reply:
looks good with just "dotnet/**"